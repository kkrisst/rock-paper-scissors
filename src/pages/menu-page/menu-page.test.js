import React from 'react'
import { shallow } from 'enzyme'

import MenuPage from './menu-page.component.jsx'

describe('Testing the MenuPage component', () => {

  const wrapper = shallow(<MenuPage />);

  it('expect MenuPage to be defined', () => {
    expect(MenuPage).toBeDefined();
  });

  it('expect MenuPage to match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('expect MenuPage to have 2 game mode buttons', () => {
    const gameModeButtons = wrapper.find('.game-modes .button');
    expect(gameModeButtons.length).toEqual(2);
  });

})