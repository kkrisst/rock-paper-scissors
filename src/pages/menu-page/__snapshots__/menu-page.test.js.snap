// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the MenuPage component expect MenuPage to match snapshot 1`] = `ShallowWrapper {}`;
